clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -45,7 +45,9 @@
   // Version 14 is the same as version 13 but it also receives a new error called OffsetMovedToTieredStorageException (KIP-405)
   //
   // Version 15 is the same as version 14 (KIP-903).
-  "validVersions": "0-15",
+  //
+  // Version 16 adds the 'NodeEndpoints' field.
```

Comment:
please include the kip as was done for the other versions

Reply:
done